### ISSUE TITLE
Documentation/Website

### ISSUE THREAD

**Issue by caelinsutch** [adham-elarabawy/OpenQuadruped#5]
This project could definitely benefit from better made documentation and a website. The wiki is also sparse in terms of setup.

**Comment by adham-elarabawy** [comment]
Agreed. I'd like to get the ROS integration done first though, since that will offload much of the setup procedure to the ROS documentation, which is actively maintained.

**Comment by caelinsutch** [comment]
Kinda want to take a crack at a logo if you're ok with that 👀

**Comment by adham-elarabawy** [comment]
Haha go for it!

**Comment by caelinsutch** [comment]
Spit out a first take:

 ![Logo](https://i.imgur.com/1CZC1e7.jpg)

Minimal vector art version of the quadraped, plus a nice pop of color with the text. Branding/website could be super cool with this color theme too.

**Comment by adham-elarabawy** [comment]
Looks great! I would actually prefer it if the text were blue. Could you do that? The preferred accent color for blue is: #55acee or some variant of that base color (slightly darker/lighter).

**Comment by caelinsutch** [comment]
> The preferred accent color for blue is: #55acee

Bam

![Logo](https://i.imgur.com/phQMccM.jpg)

**Comment by adham-elarabawy** [comment]
Oh last thing: it is spelled "quadruped" not "quadraped". Other than that, it looks perfect!

**Comment by caelinsutch** [comment]
@adham-elarabawy Sweet, you interested in a website? I'll make something with Gatsby so you can use it for documentation too :)

**Comment by adham-elarabawy** [comment]
@caelinsutch That would be great! Please let me know what you need from me.

**Comment by caelinsutch** [comment]
> @caelinsutch That would be great! Please let me know what you need from me.

I'm planning on moving over all the documentation over. It'll be a [Docusaurus 2](https://v2.docusaurus.io) site, so I'll write instructions for updating documentation and such.
